src/preferences/configobject.cpp

```diff
@@ -306,6 +347,40 @@ double ConfigObject<ConfigValue>::getValue(
     return ok ? result : default_value;
 }
 
+template<>
+template<>
+mixxx::RgbColor::optional_t ConfigObject<ConfigValue>::getValue(
```

Comment:
it might clean up this file to put `using mixxx::RgbColor` at the top of the file to reduce verbosity.

Reply:
I'd rather slowly transition all mixxx files over to the `mixxx` namespace and keep the namespace prefix until then.

Reply:
ok